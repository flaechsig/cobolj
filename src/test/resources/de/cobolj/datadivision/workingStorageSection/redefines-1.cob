IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 A   PIC 9(5) VALUE 12345.
   01 B REDEFINES A PIC 99V999.
   01 C REDEFINES A PIC 999V99.

PROCEDURE DIVISION.
   DISPLAY A.
   DISPLAY B.
   DISPLAY C.
STOP RUN.