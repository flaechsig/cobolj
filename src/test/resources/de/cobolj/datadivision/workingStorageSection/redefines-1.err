Redefines: Dezimaltrenner bei Pic9 nicht korrekt