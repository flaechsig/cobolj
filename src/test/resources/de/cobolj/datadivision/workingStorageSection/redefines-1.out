12345
12.345
123.45
